projects/canopy/src/lib/heading/heading.component.spec.ts

```diff
@@ -7,13 +7,11 @@ describe('LgHeadingComponent', () => {
   let component: LgHeadingComponent;
   let fixture: ComponentFixture<LgHeadingComponent>;
 
-  beforeEach(
-    waitForAsync(() => {
-      TestBed.configureTestingModule({
-        declarations: [ LgHeadingComponent ],
-      }).compileComponents();
-    }),
-  );
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
```

Comment:
Just wondering why `waitForAsync` is removed? I guess it's not needed in this case?